### ISSUE TITLE
Site Description column in DBEditor mostly blank

### ISSUE THREAD

**Issue by agilmore2** [opendcs/opendcs#613]
**Describe the bug**
The Public Name field in the Site record should be used to populate the DBEditor Site Description column and all Site selection dialogs instead of the "description" field, which is often blank.

**To Reproduce**
1. Open up a dialog showing sites
2. Notice that description field is blank on almost all sites

**Screenshots**
![image](https://github.com/opendcs/opendcs/assets/6852298/9ccd75d6-e4bb-41b7-8c82-b5c69c7fd691)
![image](https://github.com/opendcs/opendcs/assets/6852298/5d379194-677b-4d00-8d8f-05081e09bd16)

**Comment by MikeNeilson** [comment]
I kinda want to tweak that list to behave more like web client (e.g. no being so wide).
Since we're already doing some drastic improvements what do you think about:

1. Leave description as description
2. Add a public name column
3. Nest the "non primary" names either in a single column or under a tree view? With a drop down for change what the "primary" view is?

**Comment by MikeNeilson** [comment]
At the least I think adding a public name column, vs a behavioral check of description is missing then use public name seems like it would be better.

**Comment by agilmore2** [comment]
Some of the columns in these GUIs are controlled by the name type enum too. In my use case, public name is much more useful, and putting both may widen the display too much?

**Comment by MikeNeilson** [comment]
Yeah, that's part of why I want to collapse the "let's put every name type into a column" into a simple tree view or list within the list of a single or one extra column.
